build.gradle

```diff
@@ -7,7 +7,7 @@
  */
 
 group          = "com.etendoerp"
-version        = "1.10.0"
+version        = "1.10.1"
```

Comment:
You do not have to change the versions, this is done only by the publishing automation.